libs/design/src/molecules/navbar/navbar.md

```diff
@@ -0,0 +1,21 @@
+# Navbar
+
+The navbar is a container to be used as a navigation bar. It will align the elements horizontally within the container.
```

Comment:
The navbar is a basic element container that will align the elements horizontally within the container. It utilizes `display:flex` and expects that its direct children will take advantage of the `flexbox` model to place themselves.